RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,18 @@
+# Versioning
+
+Mlibc uses semantic versioning to denote releases.
```

Comment:
don't we stylize as `mlibc` (lowercase)?

Reply:
Mhm yeah we could